### DISCUSSION TITLE
Stellarium 0.13.2 vs. current version of Stellarium Web?

### DISCUSSION THREAD

**Discussion by Runmartinrun** [Stellarium/stellarium#1743] [Q&A]
Hello, 
I have been testing a sextant by measuring the angle between pairs of stars, and comparing the measurement with the Stellarium predictions (which include refraction corrections). 

First, I have used the  Stellarium 0.13.2, which I had installed on my laptop for a while, and the comparison was rather poor: a discrepancy of a few minutes for angles > 60 degrees. Next, I used a more recent version of Stellarium (the current one in Stellarium web) and the agreement with the same measurements greatly improved: the largest discrepancy was .3' . 

Is this somewhat expected? Are the more recent versions of stellarium more accurate than the old ones to justify such a large improvement? 

Thanks!

**Comment by gzotti** [comment]
Stellarium 0.13 is very old and should be updated for many reasons. The star catalog however is still mostly the same.  
Stellarium web is an entirely different piece of software written from scratch and with a different star catalog.
